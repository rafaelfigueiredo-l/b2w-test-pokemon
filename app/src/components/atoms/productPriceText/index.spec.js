import ProductPriceText from "./";

describe("<ProductPriceText />", () => {
  const component = <ProductPriceText>Texto</ProductPriceText>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly ProductPriceText component", () => {
    expect(wrapper.find(".productPriceText")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly ProductPriceText component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".productPriceText").children()).toHaveLength(1);
  });
});
