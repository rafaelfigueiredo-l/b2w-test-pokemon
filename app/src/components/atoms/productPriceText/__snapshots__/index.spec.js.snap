// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductPriceText /> should mount correctly ProductPriceText component 1`] = `
<styled.span
  className="productPriceText"
>
  <span
    className="sc-AxjAm fbdZIo productPriceText"
  >
    Texto
  </span>
</styled.span>
`;
